src/main/java/net/sf/jabref/importer/fileformat/BibtexParser.java

```diff
@@ -283,7 +262,7 @@ public ParserResult parse() throws IOException {
                                     + be.getAuthorTitleYear(40) + " ("
                                     + Localization.lang("grouping may not work for this entry") + ")");
                         }
-                    } catch (IOException ex) {
+                    } catch (Exception ex) {
```

Comment:
I think it would be better to find out the exact Exception (should be easy with the test) and then write a multi-catch block: `} catch (IOException | ThatOtherException ex) {`

If you catch `Exception`, you also catch Exceptions we do not want to catch, such as `NullPointerException`, `ArrayIndexOutOfBoundsException`, etc.